OPHD/States/MapViewStateTurn.cpp

```diff
@@ -509,11 +505,13 @@ void MapViewState::nextTurn()
 
 	mPreviousMorale = mCurrentMorale;
 
+	produceFood();
 	transferFoodToCommandCenter();
 	updateResidentialCapacity();
 
 	countFood();
 	updatePopulation();
+	countFood();
```

Comment:
`updatePopulation` reduces the food amount, so re-counting here to make the GUI show the correct level.

Reply:
Good catch.